File: Eating_aos/app/src/main/java/desla/aos/eating/data/model/LoginResponse.kt
package desla.aos.eating.data.model


import com.google.gson.annotations.SerializedName

data class LoginResponse(
    @SerializedName("data")
    val `data`: Data,
    @SerializedName("message")
    val message: String,
    @SerializedName("status")
    val status: String
) {
    data class Data(
        @SerializedName("address")
        val address: String,
        @SerializedName("latitude")
        val latitude: Double,
        @SerializedName("longitude")
        val longitude: Double
    )
}

File: Eating_ios/Eating/Models/LoginResponse.swift
//
//  LoginResponse.swift
//  Eating
//
//  Created by 코드잉 on 2021/03/29.
//

import Foundation

class PostResponse: Codable {
    var data: AddressResponse
}

class AddressResponse: Codable {
    var address: String
    var latitude: Double
    var longitude: Double
}
